types/networks/src/lib.rs

```diff
@@ -302,3 +304,68 @@ impl Default for ChainConfig {
         }
     }
 }
+
+pub fn default_network_version() -> NetworkVersion {
+    NetworkVersion::V15
```

Comment:
Unless we have good reasons to, I think it would be better to have `NetworkVersion::V1` here.